metron-platform/metron-common/src/test/java/org/apache/metron/common/stellar/StellarTest.java

```diff
@@ -593,6 +595,39 @@ public void testStringFunctions_advanced() throws Exception {
   }
 
   @Test
+  public void testLeftRightFills() throws Exception{
+    final Map<String, Object> variableMap = new HashMap<String, Object>() {{
+      put("foo", null);
+      put("bar", null);
+      put("notInt","oh my");
+    }};
+
+    Object left = run("FILL_LEFT('123','X', 10)",new HashedMap());
+    Object right = run("FILL_RIGHT('123','X', 10)", new HashedMap());
+    Object same = run("FILL_RIGHT('123','X', 3)", new HashedMap());
+    Object nullIn = run("FILL_RIGHT('123',foo,bar)", variableMap);
+    Object notInt= run("FILL_RIGHT('123',foo, notInt)", variableMap);
```

Comment:
Would it make sense to add more tests for the failure behavior?  Like nulls for the length and fill arguments.  Also the case where the actual length is greater than the fill length. There's probably some other corner cases we could hit too.